exec/java-exec/src/main/java/org/apache/drill/exec/physical/base/AbstractGroupScanWithMetadata.java

```diff
@@ -151,13 +152,13 @@ public long getColumnValueCount(SchemaPath column) {
     long colNulls;
     if (columnStats != null) {
       Long nulls = (Long) columnStats.getStatistic(ColumnStatisticsKind.NULLS_COUNT);
-      colNulls = nulls != null ? nulls : GroupScan.NO_COLUMN_STATS;
+      colNulls = nulls != null ? nulls :  Statistic.NO_COLUMN_STATS;
```

Comment:
Please remove extra space here:
```suggestion
      colNulls = nulls != null ? nulls : Statistic.NO_COLUMN_STATS;
```